x-pack/plugins/cases/public/components/app/routes.test.tsx

```diff
@@ -46,70 +46,63 @@ const renderWithRouter = (
   );
 };
 
-describe('Cases routes', () => {
-  describe('All cases', () => {
-    it('navigates to the all cases page', () => {
-      renderWithRouter();
-      expect(screen.getByText('All cases')).toBeInTheDocument();
-    });
+for (let i = 0; i <= 200; i = i + 1) {
+  describe('Cases routes', () => {
+    describe('All cases', () => {
+      it('navigates to the all cases page', async () => {
+        renderWithRouter();
+        expect(await screen.findByText('All cases')).toBeInTheDocument();
+      });
 
-    // User has read only privileges
-    it('user can navigate to the all cases page with only read permissions', () => {
-      renderWithRouter(['/cases'], readCasesPermissions());
-      expect(screen.getByText('All cases')).toBeInTheDocument();
+      // User has read only privileges
+      it('user can navigate to the all cases page with only read permissions', async () => {
+        renderWithRouter(['/cases'], readCasesPermissions());
+        expect(await screen.findByText('All cases')).toBeInTheDocument();
+      });
     });
-  });
-
-  // FLAKY: https://github.com/elastic/kibana/issues/163263
-  describe.skip('Case view', () => {
-    it.each(getCaseViewPaths())(
-      'navigates to the cases view page for path: %s',
-      async (path: string) => {
-        renderWithRouter([path]);
-        await waitFor(() => {
-          expect(screen.getByTestId('case-view-loading')).toBeInTheDocument();
-        });
-
-        // User has read only privileges
-      }
-    );
 
-    it.each(getCaseViewPaths())(
-      'user can navigate to the cases view page with read permissions and path: %s',
-      async (path: string) => {
-        renderWithRouter([path], readCasesPermissions());
-        await waitFor(() => {
-          expect(screen.getByTestId('case-view-loading')).toBeInTheDocument();
-        });
-      }
-    );
-  });
+    // FLAKY: https://github.com/elastic/kibana/issues/163263
+    describe.skip('Case view', () => {
+      it.each(getCaseViewPaths())(
+        'navigates to the cases view page for path: %s',
```

Comment:
didn't work :(

Reply:
As we talked offline checking for the `'case-view-loading'` does not feel right. We should mock the `CaseView` as we do for the `AllCases` on the top of the file.

Reply:
I was trying to do what you suggested and couldn't get it to work probably because `CaseView` is lazy loaded in the `CasesRoutes` component. [[see here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/components/app/routes.tsx#L83)]

None of the other mocked components are lazy loaded and I think the issues I was having now are probably why this test only looked for `'case-view-loading'`.

Alternatively, I mock `useGetCases` to make sure `isError` and `isLoading` are always `false`, and then mocked the `CaseViewPage` component like the other on top of the file. This makes sure the CaseView always renders something. [[see here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/cases/public/components/case_view/index.tsx#L80)]

Locally it works, let's see here with the loop.